uaclient/cli/tests/test_cli_attach.py

```diff
@@ -270,46 +288,13 @@ def fake_contract_attach(contract_token, attachment_dt):
         ret = action_attach(args, cfg)
 
         assert 0 == ret
-        assert 1 == m_status.call_count
-        assert 1 == m_format_tabular.call_count
         expected_calls = [
             mock.call(contract_token=token, attachment_dt=mock.ANY)
         ]
         assert expected_calls == contract_machine_attach.call_args_list
         assert [mock.call(cfg)] == m_update_apt_and_motd_msgs.call_args_list
         assert 1 == m_update_activity_token.call_count
-
-        # We need to do that since all config objects in this
-        # test will share the same data dir. Since this will
-        # test a successful attach, in the end we write a machine token
-        # file, which will make all other cfg objects here to report
-        # as attached
-        cfg.delete_cache()
-        cfg.machine_token_file.delete()
-
-        cfg = FakeConfig()
-        args = mock.MagicMock(token=token, attach_config=None)
-        with mock.patch.object(
-            event, "_event_logger_mode", event_logger.EventLoggerMode.JSON
-        ):
-            with mock.patch.object(
-                cfg, "check_lock_info"
-            ) as m_check_lock_info:
-                m_check_lock_info.return_value = (0, "lock_holder")
-                fake_stdout = io.StringIO()
-                with contextlib.redirect_stdout(fake_stdout):
-                    main_error_handler(action_attach)(args, cfg)
-
-        expected = {
```

Comment:
With this new setup we are no longer testing the `event_logger` part of the command. Should we add an explicit test on for this as well ?

Reply:
We have integration tests that cover json output for attach, so I think this is okay to remove.